src/CreateHandler/CreateHandler.php

```diff
@@ -37,10 +37,11 @@ public function __construct(string $classSkeleton = self::CLASS_SKELETON, string
     }
 
     /**
+     * @param string $class
      * @param array $additionalSubstitutions An associative array where the keys
      *     are the substitution strings to match, and the values are the associated
      *     values to substitute.
-     * @throws CreateHandlerException
+     * @return string
```

Comment:
The same about `@return` tag. We have defined RTH with the method declaration so it's useless.